### ISSUE TITLE
How does this work?

### ISSUE THREAD

**Issue by PlaceScore** [pvandegeer/GeometryShapes#9]
I have just installed the plugin. I now see a new menu item under _Vector > Geometry Shapes_ but both the submenu items are disabled (_Draw rectangle geometry_, _Draw oval geometry_).

How does this plugin work?

**Comment by pvandegeer** [comment]
Do you have an _editable_ later with _polygon_ geometries?

**Comment by PlaceScore** [comment]
> Do you have an _editable_ later with _polygon_ geometries?

No. I have a map layer which looks fine, and want to overlay some circles over a few cities. How do I do that?

**Comment by pvandegeer** [comment]
You need to create a layer for the circles to be drawn on. 
If you just want to draw circles for printing or making images you might be better of doing that in the print composer.

**Comment by PlaceScore** [comment]
> You need to create a layer for the circles to be drawn on. If you just want to draw circles for printing or making images you might be better of doing that in the print composer.

Thanks. It's pretty good! Just goes to show how a small tutorial would help. See also https://gis.stackexchange.com/questions/479366/overlay-a-circle-in-qgis where I was hunting for an alternative solution since I didn't hear back from you.
